### DISCUSSION TITLE
Default PFT parameters?

### DISCUSSION THREAD

**Discussion by Aariq** [EDmodel/ED2#376] [Q&A] [answered]
Where (in the wiki or in the code) would I go to find the default values for PFT parameters(https://github.com/EDmodel/ED2/wiki/PFT-parameters)?

**Comment by xiangtaoxu** [comment, marked as answer]
Hi, Eric. You should be able to find all PFT parameters in [src/init/ed_params.f90](https://github.com/EDmodel/ED2/blob/master/ED/src/init/ed_params.f90)

Meanwhile some derived parameters might not be easy to figure out by just looking at ed_params.f90. I often do a 1-day simulation incluidng the PFTs you are interested in. Then find the output .xml file which will record all the parameters.

**Reply by Aariq** [reply]
Thanks!
